src/VisualStudio/Setup/source.extension.vsixmanifest

```diff
@@ -4,13 +4,33 @@
     <Identity Id="0b5e8ddb-f12d-4131-a71d-77acc26a798f" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Language Services</DisplayName>
     <Description>C# and VB.NET language services for Visual Studio.</Description>
+    <ShortcutPath>..\CommonExtensions\Microsoft\ManagedLanguages\VBCSharp\LanguageServices</ShortcutPath>
   </Metadata>
   <Installation Experimental="true">
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.Pro" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VSWinDesktopExpress" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VWDExpress" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VSWinExpress" />
   </Installation>
+  <Installer>
+    <Actions>
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.EditorFeatures.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.Features.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.Workspaces.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.EditorFeatures.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.EditorFeatures.Text.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.Features.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.VisualBasic.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.VisualBasic.EditorFeatures.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.VisualBasic.Features.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.VisualBasic.Workspaces.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.Workspaces.Desktop.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.Workspaces.dll" />
+      <Action Type="Ngen" Path="Microsoft.VisualStudio.LanguageServices.dll" />
```

Comment:
Oh.  Hmm, this might be a problem.  We currently get around that by having a list of binaries not to generate setup authoring for.  I believe we want them in the vsixes for dogfooding when we update them, but not in the vsix as part of a standard install.

@tmat probably knows more about this part.